runtime/vm/createramclass.cpp

```diff
@@ -2785,7 +2788,9 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 
 	if (J2SE_VERSION(javaVM) >= J2SE_19) {
 		if (NULL == classBeingRedefined) {
-			if (J9_ARE_ALL_BITS_SET(javaVM->runtimeFlags, J9_RUNTIME_JAVA_BASE_MODULE_CREATED)) {
+			if (J9ROMCLASS_IS_ARRAY(romClass)) {
```

Comment:
Why is this check before the check if java.base is created?

Reply:
Element classes are always loaded before array classes, so at this it doesn't matter if java.base is created or not because we will take the module from the element class.

Reply:
Can you add a comment to that effect to the code?  Makes it obvious to the next reader and leaves a breadcrumb for yourself / others.

Reply:
I added a comment